### PR TITLE
Add BuildType to Github Repo Pages

### PR DESCRIPTION
**Pull request by brandonstubbs**
resolves: #2723

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @brandonstubbs !
LGTM.

Awaiting LGTM+Approval from any other contributor to this repo before merging.

**Review by Sairaviteja27 — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @Sairaviteja27 !
Merging.

**Pull request merged**
